presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -1134,6 +1135,17 @@ public Void visitShowGrants(ShowGrants node, Integer indent)
             return null;
         }
 
+        @Override
+        public Void visitSetPath(SetPath node, Integer indent)
+        {
+            builder.append("SET PATH ");
```

Comment:
`format` call is redundant, `map` call can be simplified to use method references:

```
builder.append(node.getPathSpecification().getPath().stream()
        .map(PathElement::toString)
        .collect(joining(", ")));
```

Reply:
The changed code is one line beneath the code referenced in this comment.